contrib/guix/README.md

```diff
@@ -261,6 +261,7 @@ details.
 
   - `guix` build commands as in `guix shell --cores="$JOBS"`
   - `make` as in `make --jobs="$JOBS"`
+  - `cmake` as in `cmake --build build -j "$JOBS"`
```

Comment:
do we need both in guix?

Reply:
GNU Make is still used to build depends.